### DISCUSSION TITLE
[Feature] Allow to merge/hide facial detection entries

### DISCUSSION THREAD

**Discussion by ToxicMushroom** [immich-app/immich#2470] [Feature Request]
### The feature

Thanks for adding these great features <3 !

Some initial issues i found when trying it out:
 - Sometimes the facial detection catches some random background person.
 - And sometimes it creates multiple entries for the same person due to different lighting or blur. 

Maybe some manual merging/hiding could be added.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by xyxzxyz** [comment]
We def need this!

**Comment by jrasm91** [comment]
Great ideas. Let's track/discuss this in #2472
